linux/shim/init.go

```diff
@@ -106,6 +106,15 @@ func (p *initProcess) Status() int {
 	return p.status
 }
 
+// ContainerStatus returns e.g. "running"
```

Comment:
nit: `// ContainerStatus return the state of the container (created, running, paused, stopped`